### PR TITLE
should fix 131

### LINKED ISSUE #131: Change default city in both volunteer registration and help request to be blank and force people to choose

### PR DESCRIPTION
**Pull request by BigBabik**
Resolves #131

### CODE REVIEW

File: client/forms.py [hunk 1]
```diff
@@ -54,6 +55,8 @@ class VolunteerForm(forms.Form):
     )
 
     my_validator = RegexValidator(r"^\d+$")
+    not_blank_validator = RegexValidator(r"^(?!\s*$).+")
```
Comment (itay-sho) [inline client/forms.py]:
```suggestion
    not_blank_validator = RegexValidator(r"^.+$")
```
would be simpler

File: client/forms.py [hunk 2]
```diff
@@ -20,6 +20,7 @@
 onlyNames = [a["name"] for a in data]
 onlyNames.sort()
 CITIES = [(str(x), str(x)) for x in onlyNames]
+CITIES.append(('', ''))
```
Comment (itay-sho) [inline client/forms.py]: shouldn't it be the 1st?
plus, add a comment that explains it.

File: client/forms.py [hunk 3]
```diff
@@ -159,11 +162,13 @@ class BaseHelpForm(forms.Form):
         ('OTHER', 'אחר')
     )"""
 
-    my_validator = RegexValidator(r"^\+?(972|0)(\-)?0?(([23489]{1}\d{7})|[5]{1}\d{8})$")
+    phone_validator = RegexValidator(r"^\+?(972|0)(\-)?0?(([23489]{1}\d{7})|[5]{1}\d{8})$")
+    not_blank_validator = RegexValidator(r"^(?!\s*$).+")
```
Comment (itay-sho) [inline client/forms.py]: this validator is used more than once, you should have a constant that holds it

File: client/forms.py (hunk 2 above)
Comment (BigBabik) [reply client/forms.py:23]: Why should it be first? I don't think it matters

Comment (astar26) [reply client/forms.py:27]: Users should only search one way of the list, preferably top-down (more intuitive).

File: client/forms.py [hunk 4]
```diff
@@ -15,11 +15,16 @@
...
+NOT_BLANK_VALIDATOR = RegexValidator(r"^.+$")
+
 json_file = open('./client/city.json', encoding="utf-8")
 data = json.load(json_file)
 onlyNames = [a["name"] for a in data]
 onlyNames.sort()
 CITIES = [(str(x), str(x)) for x in onlyNames]
+
+# Add a blank option to CITIES in order to enable setting the initial value of the ChoiceField to ''.
+CITIES.append(('', ''))
```
Comment (itay-sho) [inline client/forms.py:27]:
```suggestion
CITIES = [('', '')] + CITIES
```

**Pull request merged**